services/frontend-service/src/ui/components/ServiceLane/ServiceLane.scss

```
@@ -62,6 +62,7 @@ Copyright freiheit.com*/
         display: flex;
         margin: $service-lane-releases-margin;
         flex-flow: wrap;
+        min-height: 85px;
```

Make sure that the loading service lane has the same height as the ready one

---

this should be a comment in the code
```suggestion
        min-height: 85px; // Ensure that the loading service lane has the same height as the ready one
```